ONSdigital/eq-survey-runner
# Fix relaunch after adding visitor

### What is the context of this PR?
Fixes a bug where, after signing out once completing the any visitors question, users would enter an infinite redirects situation.

### How to review 
Follow these steps:

1. Launch the census household survey
2. Add three people to the household
3. Add a visitor to the household
4. Sign out
5. Launch the survey with the same response id
6. Ensure you are redirected to the who lives here section summary page.
